tests/deploy/ui/test_ui.py

```diff
@@ -61,8 +60,8 @@ def start(self):
             time.sleep(1)
 
     def stop(self):
-        self.proc.kill()
-        pn.state.kill_all_servers()
+        self.proc.terminate()
```

Comment:
We cannot use `proc.kill()` here, because a process cannot handle the signal as it is a "last resort". Graceful shutdown is achieved by `proc.terminate()`.